docs/source/conf.py

```diff
@@ -0,0 +1,49 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'sncli rtfd'
```

Comment:
I'd rather this project info was kept consistent with the project itself. `sncli` and the same copyright information as found in https://github.com/insanum/sncli/blob/master/simplenote_cli/__init__.py#L3

I'm not sure what the copyright notice should be since there have been several major contributors to this project now. maybe we should add something like "and contributors" sometime. I'll do some research.

Reply:
This can all be changed. I just put in something to get started with.

Reply:
Cool no worries :+1: